src/realm/sync/noinst/client_impl_base.hpp

```diff
@@ -328,25 +324,22 @@ class ClientImpl {
     // approach would be to allow for per-endpoint SSL parameters to be
     // specifiable through public member functions of ClientImpl from where they
     // could then be picked up as new connections are created on demand.
-    //
-    // FIXME: `session_multiplex_ident` should be eliminated from ServerEndpoint
```

Comment:
i'm not even sure what this meant.

Reply:
If we set `one_connection_per_session = false` and then assigned each session a unique session_multiplex_ident we'd end up with a separate connection per session but tracked somewhat differently from when `one_connection_per_session = true` (one ServerSlot per connection rather than multiple connections in one ServerSlot), and this resulted in reconnect delays behaving differently..